internal/website/content/howto/server/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Server"
+date: 2019-06-21T10:36:43-07:00
+draft: true
+showInSidenav: true
+---
+
+The Go CDK's `server` package provides a pre-configured HTTP server with 
+diagnostic hooks for request logging, health checks, and trace exporting via 
+OpenCensus. This guide will show you how to start up and shut down the server,
+as well as how to work with the request logging and health checks.
+
+## Starting up the server
+
+The Go CDK Server constructor takes an `http.Handler` and an `Options` struct. 
+The simplest way to start the server is to use `http.DefaultServeMux` and
+pass `nil` for the options.
+
+{{< goexample src="gocloud.dev/server.ExampleServer_New" >}}
+
+### Adding a request logger
+
+You can use the `server.Options` struct to specify a request logger.
+
+The example is shown with the Go CDK [`requestlog`](https://godoc.org/gocloud.dev/requestlog) package's `NCSALogger`.
+To get logs in the Stackdriver JSON format, use `NewStackdriverLogger` in place
+of `NewNCSALogger`.
+
+{{< goexample src="gocloud.dev/server.ExampleServer_RequestLogger" >}}
+
+### Adding health checks
+
+The Go CDK `server` package affords a hook for you to define health checks for
+your application and see the results at `/healthz/readiness`. The server also
+runs an endpoint at `/healthz/liveness` which will always return `HTTP 200` if
+the server is up, which is also the endpoint Kubernetes will use to check liveness.
+
+Health checks are an important part of application monitoring, and readiness
+checks are subtly different than liveness checks. The liveness check will always
```

Comment:
"always and simply" sounds a bit odd, maybe "will always return" or "will simply return"?

Reply:
Or just drop this sentence, you said it above?